src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    padding-top: 40px;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+  }
+
+  &__code {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    padding-top: 5px;
+    color: #616070;
+  }
+
+  &__stars {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 17px;
+  }
+
+  &__stars_reviews {
+    display: flex;
+    align-items: center;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 16px;
+    padding-top: 24px;
+  }
+
+  &__price_text {
+    display: flex;
+    align-items: center;
+    font-weight: 400;
+    font-size: 12px;
+    color: #616070;
+  }
+
+  &__price-cost {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    width: 166px;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Use paddings instead

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/222683048-bc162606-fdc5-4bcd-a13d-626749aff282.png)

Reply:
Try to fix this. If you have any questions - feel free to ask them in `fe_chat`.